tools/checkpoint_conversion/convert_roberta_checkpoints.py

```diff
@@ -42,16 +44,22 @@
 )
 
 
-def download_model(preset, size, hf_model_name):
+def download_model(size, hf_model_name):
     print("-> Download original weights.")
     extract_dir = EXTRACT_DIR.format(size)
     archive_file_path = keras.utils.get_file(
         fname=None,
         origin=DOWNLOAD_SCRIPT_URL.format(size),
-        cache_subdir=os.path.join("checkpoint_conversion", preset),
+        cache_subdir=os.path.join("checkpoint_conversion", FLAGS.preset),
     )
 
-    os.system(f"tar -xvf {archive_file_path}")
+    archive_file_path = os.path.abspath(archive_file_path)
+    if archive_file_path.endswith(".tar.gz"):
+        with tarfile.open(archive_file_path, "r:gz") as tar:
+            tar.extractall(extract_dir)
+    elif archive_file_path.endswith(".zip"):
+        with zipfile.ZipFile(archive_file_path, "r") as zip_ref:
+            zip_ref.extractall(extract_dir)
```

Comment:
Let's shift this to a function inside https://github.com/keras-team/keras-nlp/blob/master/tools/checkpoint_conversion/checkpoint_conversion_utils.py.